tagging/create_manifests.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+import argparse
+import datetime
+import logging
+import os
+from docker_runner import DockerRunner
+from get_taggers_and_manifests import get_taggers_and_manifests
+from git_helper import GitHelper
+from taggers import SHATagger
+
+
+logger = logging.getLogger(__name__)
+
+
+BUILD_TIMESTAMP = datetime.datetime.utcnow().isoformat()[:-7] + "Z"
+MARKDOWN_NEWLINE = "<br />"
+
+
+def append_build_history_line(short_image_name, owner, wiki_path, all_tags, container):
+    date_column = f"`{BUILD_TIMESTAMP}`"
+    image_column = MARKDOWN_NEWLINE.join(
+        f"`{owner}/{short_image_name}:{tag_value}`" for tag_value in all_tags
+    )
+    commit_sha_tag = SHATagger.tag_value(container)  # first 12 letters of commit hash
+    commit_hash = GitHelper.commit_hash()  # full commit hash
```

Comment:
I have no idea why we use only 12 letters for tagging.
It makes the code a bit more difficult.

But, for now, I don't want to change the behavior (it would be too difficult, but in the future, it might be easier to use full hash as a tag).